src/data/polynomial/basic.lean

```diff
@@ -793,9 +775,9 @@ variables [ring R]
 instance : has_int_cast R[X] := ⟨λ n, of_finsupp n⟩
 
 instance : ring R[X] :=
-function.injective.ring to_finsupp to_finsupp_injective
-  to_finsupp_zero to_finsupp_one to_finsupp_add to_finsupp_mul to_finsupp_neg to_finsupp_sub
-  (λ _ _, to_finsupp_smul _ _) (λ _ _, to_finsupp_smul _ _) to_finsupp_pow (λ _, rfl) (λ _, rfl)
+to_finsupp_injective.ring to_finsupp to_finsupp_zero to_finsupp_one to_finsupp_add to_finsupp_mul
```

Comment:
Is this just a style change?

Reply:
again, I changed this line while playing with the PR and then it ended up similar to how it started, I can revert this if needed.